pkg/pkgmgr/dpkg.go

```diff
@@ -344,8 +344,12 @@ func (dm *dpkgManager) installUpdates(ctx context.Context, updates unversioned.U
 		llb.IgnoreCache,
 	).Root()
 
-	checkUpgradable := `sh -c "apt-get -s upgrade 2>/dev/null | grep -q "^Inst" || exit 1"`
+	checkUpgradable := `sh -c "apt-get -s upgrade 2>/dev/null | grep -q "^Inst" > /upgradable.txt || exit 1"`
```

Comment:
can we do the same thing in the case of distroless images (unpackAndMergeUpdates func)

Reply:
and then same thing for rpm distroless just to keep the errors consistent